test/struct.js

```diff
@@ -379,4 +379,29 @@ describe('Struct', function () {
 
   })
 
+  describe('usePack()', function () {
+    var size = {
+        unpack: { ia32: 8, x64: 16 },
```

Comment:
There's also `arm`, and possible others. Maybe we should do an `is64Bit` check by checking `ref.sizeof.pointer === 8` and then fall back to 32-bit if not.